index.bs

```diff
@@ -1623,9 +1626,10 @@ a \[[stream]] internal slot. When a <code>WritableStream</code> error function <
     1. Resolve _writeRecord_.[[promise]] with *undefined*.
     1. DequeueValue(_stream_@[[queue]]).
     1. Let _syncResult_ be SyncWritableStreamStateWithQueue(_stream_).
-    1. If _syncResult_ is an abrupt completion, then return ErrorWritableStream(_stream_, ‍_syncResult_.[[value]]).
-    1. Otherwise, return WritableStreamAdvanceQueue(_stream_).
-  1. Upon rejection of _writeResult_ with reason _r_, return ErrorWritableStream(_stream_, _r_).
+    1. If _syncResult_ is an abrupt completion, then perform ErrorWritableStream(_stream_, ‍_syncResult_.[[value]]).
```

Comment:
Good catches both. For GetTotalQueueSize() and other operations with return values that we need to use, I am just going to omit the ReturnIfAbrupt. Maybe it would be better to do something like AssertNotAbrupt, but I think it is OK to just start using the value.

Fixing and pushing.